package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@maarkllc/code-config",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Since this PR introduces a breaking change – mostly because the configurations structure changes – I wonder if I should change this to `2.0.0` instead.